Workbooks/Virtual Machines - Performance Analysis/Performance Analysis for a Group of VMs/Performance Analysis for a Group of VMs.workbook

```diff
@@ -285,7 +253,7 @@
             "name": "Test",
             "type": 1,
             "isRequired": false,
-            "query": "// {DefaultWorkspace} {ContextFree} {ContextSelection} {RandomSelection} {Selection} {Subscriptions} {Workspaces}\r\nPerf\r\n| take 1",
+            "query": "Perf\r\n| take 1",
```

Comment:
these queries don't seem to use time range, either in the query or in the time range picker, is that ok?  this query will be limited to last 24h, is that ok?

Reply:
I think for `Test` parameter we don't want to limit it by any time range but search the entire `Perf` table for any entries (or if the table even exists, at which point the parameter will be empty since it's marked as not required). I didn't understand the second part, are you saying that if I don't include a time range that it will default to the last 24 hours? Because that is not okay since that may give the wrong prompt to the user. I set the Time Range to `Set in query` so my assumption was that it would search the entire table regardless of time range.

![image](https://user-images.githubusercontent.com/43890980/58142348-83d08780-7bfb-11e9-8d87-23ebeae927da.png)

Reply:
if it is set to "set in query", and the query doesn't *explicitly* set a range, then yes, it defaults to 24h.  otherwise, that query could be looking at YEARS of data.  this is the same behavior as the log analytics portal's query stuff.  if you don't specify a time range, you get 24h.  it's always best to set a time range either explicitly in the query text or by picking the parameter.

Reply:
Thank you @gardnerjr, I have updated the `Test` parameter and the corresponding help text to ask the user to now either adjust the time range or select a different workspace, along with onboarding instructions. Also I've added an error text because I still see cases where the `Subscription` parameter fails to initialize so I ask the user to take corrective action as well.